mimic/rest/identity_admin_api.py

```diff
@@ -0,0 +1,102 @@
+# -*- test-case-name: mimic.test.test_identity_admin -*-
+"""
+Mocks for the identity admin API.
+"""
+from json import dumps, loads
+
+from zope.interface import implementer
+from twisted.plugin import IPlugin
+
+from mimic.imimic import IAPIMock
+from mimic.rest.mimicapp import MimicApp
+from mimic.model.identity_admin_objects import bad_request
+
+
+@implementer(IAPIMock, IPlugin)
```

Comment:
http://developer.openstack.org/api-ref-identity-admin-v2.html strongly suggests that the identity admin API is co-located with the Identity API and therefore would not, itself, show up in the service catalog.  `IAPIMock` plugins are plugins which show up in the service catalog, so this stuff would definitely have to go in the place with the rest of the Identity routes, i.e. `mimic.rest.auth_api`.